### ISSUE TITLE
Page title in GOV.UK Frontend review application is more verbose than it needs to be

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1412]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue28
**URL**: http://govuk-frontend-v3.herokuapp.com/full-page-examples/feedback
**Page**: Feedback

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58715885-b75f9e80-83bf-11e9-84ca-09bee07011d3.png)

The page titles were descriptive of the pages tested; however, the context of the page was given after GOV.UK. This means that it takes longer for a screen reader to determine the purpose of the page and results in the context being truncated for sighted users.

## Current Code Ref(s)
```html
<title>GOV.UK - Send your feedback to GOV.UK Verify</title>
```

## Screen reader comments
> “The page titles throughout the service were either not front loaded or did not contain the purpose of the page. For a live service this could mean that the page titles do not reflect the GDS style guidelines which indicate that the page title format should be the page purpose followed by the service name and a .gov ending. This occurred for all pages tested with JAWS, NVDA and VoiceOver for iPhone and Mac.”

## Cognitive users’ comments
> “I noticed that the page title was cut off on some pages and I could not read it unless I hovered my mouse over it.”

## Solution
Ensure that page titles are ‘front-loaded’ with the name of the page appearing prior to the service name and GOV.UK text.
Example: `Page title – service name – GOV.UK`

**Comment by nickcolley** [comment]
I have assigned this as low priority since it is not an issue with GOV.UK Frontend but instead with our implementation of the review application pages.

However it would not take long to update so we should fix this.

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
